bin/MadGraph5_aMCatNLO/cards/WW_leptonic/WWjj_SS_dim8_ewk_qcd_extramodels.dat

```diff
@@ -0,0 +1,2 @@
+SM_Ltotal_Ind5v2020v2_UFO.tar.gz
```

Comment:
https://cms-project-generators.web.cern.ch/cms-project-generators/SM_Ltotal_Ind5v2020v2_UFO.zip
Isn't that the correct model ?